src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/JiraSoftwareDevDistribution.kt

```diff
@@ -0,0 +1,42 @@
+package com.atlassian.performance.tools.awsinfrastructure.api.jira
+
+import com.atlassian.performance.tools.awsinfrastructure.api.storage.S3Artifact
+import com.atlassian.performance.tools.awsinfrastructure.api.storage.S3TarGzDistribution
+import com.atlassian.performance.tools.infrastructure.api.distribution.ProductDistribution
+import com.atlassian.performance.tools.ssh.api.SshConnection
+import java.time.Duration
+
+/**
+ * Distributes dev versions, e.g. Jira built from source code.
+ * Requires S3 credentials, therefore it's unsuitable for non-Atlassian users.
```

Comment:
Still good to see it open source (as it's another example of ProductDistribution implementation) 👍.